homeassistant/components/elgato/quality_scale.yaml

```diff
@@ -33,7 +33,10 @@ rules:
   entity-unavailable: done
   integration-owner: done
   log-when-unavailable: done
-  parallel-updates: done
+  parallel-updates:
+    status: todo
+    comment: |
+      Does not set parallel-updates on button/switch action calls.
```

Comment:
For the other components, I have set `PARALLEL_UPDATES = 0` as a coordinator is used to centralize the data updates
However for `elgato` it is missing on platforms with action calls so I chose to downgrade the status instead.

Reply:
The question for me is rather: Should we disregard read-only platforms entirely when a coordinator is used?

Reply:
There's been a lot of questions on Discord about when this rule applies.
IMO, explicit is better than implicit and reduces the burden on the reviewer.

Reply:
I'm fine with that, but we should then update the rule docs to state that clearly as well.

Reply:
> I'm fine with that, but we should then update the rule docs to state that clearly as well.

https://github.com/home-assistant/developers.home-assistant/pull/2482